.devcontainer/Dockerfile

```diff
@@ -0,0 +1,51 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
You might want to change that copyright notice?

Reply:
Hmmm, it is modified from Microsoft's suggested Dockerfile though, and I think it has to be propagated, as per the license in the URL. Specifically:

>     The above copyright notice and this permission notice shall be included in all
>     copies or substantial portions of the Software.

Reply:
We then keep it I guess, but maybe link to the original?

Reply:
Done!